model_utils/tracker.py

```diff
@@ -45,7 +45,10 @@ def current(self, fields=None):
     def has_changed(self, field):
         """Returns ``True`` if field has changed from currently saved value"""
         if field in self.fields:
-            return self.previous(field) != self.get_field_value(field)
+            try:
+                return self.previous(field) != self.get_field_value(field)
+            except TypeError:
+                return True
```

Comment:
In our current issue situation, it will mask the `TypeError`, which will be even harder to detect. I think we should properly handle timezone related issues.